DAO.sol

```diff
@@ -586,7 +590,6 @@ contract DAO is DAOInterface, Token, TokenCreation {
             // related addresses. Proxy addresses should be forbidden by the curator.
             if (p.recipient != address(this) && p.recipient != address(rewardAccount)
                 && p.recipient != address(DAOrewardAccount)
-                && p.recipient != address(extraBalance)
                 && p.recipient != address(curator)) {
```

Comment:
Do you think this should be removed: `&& p.recipient != address(extraBalance)`? If we remove it I think we will be double counting reward tokens.

What we should do (I think) is to move the balance from the extraBalance to the DAO's balance without creating reward tokens initially. Then once those fund are used to fund a real proposal, the reward token balance will be incremented appropriate.







I think we still want that check `&& p.recipient != address(extraBalance)` because we don't want to allow the recipient of DAO funds to be the extraBalance managed account itself (or other DAO related accounts).

So to move funds to the 

extraBalance